README.md

```diff
@@ -1,86 +1,44 @@
 # Puppet-contrail
 
-[![Build Status](https://travis-ci.org/redhat-cip/puppet-contrail.png?branch=master)](https://travis-ci.org/redhat-cip/puppet-contrail)
-[![Puppet Forge](http://img.shields.io/puppetforge/v/eNovance/contrail.svg)](https://forge.puppetlabs.com/eNovance/contrail)
 [![License](http://img.shields.io/:license-apache2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
 
 #### Table of Contents
 
-1. [Overview - What is the puppet-contrail module?](#overview)
-2. [Module Description - What does the module do?](#module-description)
-3. [Setup - The basics of getting started with puppet-contrail](#setup)
-4. [Implementation - An under-the-hood peek at what the module is doing](#implementation)
-5. [Limitations - OS compatibility, etc.](#limitations)
-6. [Development - Guide for contributing to the module](#development)
-7. [Contributors - Those with commits](#contributors)
-
-Overview
---------
-
-A [Puppet Module](http://docs.puppetlabs.com/learning/modules1.html#modules) is a collection of related content that can be used to model the configuration of a discrete service.
+1. [Module Description - What does the module do?](#module-description)
+2. [Setup OpenStack Cloud with Contrail Networking](#setup-openstack-cloud-with-contrail-networking)
 
 Module Description
 ------------------
 
-The puppet-contrail module is a thorough attempt to make Puppet capable of managing the entirety of OpenContrail.  This includes manifests to provision region specific endpoint and database connections. Types are shipped as part of the puppet-contrail module to assist in manipulation of configuration files.
-
-Setup
------
-
-**What the puppet-contrail module affects**
-
-* puppet-contrail, the Juniper SDN service.
-
-### Installing puppet-contrail
-
-    example% puppet module install enovance/contrail
+The puppet-contrail module is a thorough attempt to make Puppet capable of managing the entirety of OpenContrail. The current implementation is limited to support Contrail v4.0 and to be used together with [RDO](https://www.rdoproject.org/), [Contral TripleO Heat Templates](https://github.com/Juniper/contrail-tripleo-heat-templates) and [Contrail TripleO Puppet](https://github.com/Juniper/contrail-tripleo-puppet).
```

Comment:
I see $version used within the puppet manifests. why does README say "The current implementation is limited to support Contrail v4.0"?